sunpy/visualization/imageanimator.py

```diff
@@ -949,20 +952,20 @@ def __init__(self, data, wcs=None, image_axes=[-1, -2], unit_x_axis=None, unit_y
         self.slices_wcsaxes = list_slices_wcsaxes[::-1]
         self.unit_x_axis = unit_x_axis
         self.unit_y_axis = unit_y_axis
-        super(ImageAnimatorWCS, self).__init__(data, image_axes=image_axes, axis_ranges=axis_ranges, **kwargs)
+        super(ImageAnimatorWCS, self).__init__(data, image_axes=image_axes, unit_x_axis, unit_y_axis , axis_ranges=axis_ranges, **kwargs)
```

Comment:
Here the order is broken.

Reply:
But the relative ordering is same as before it was updated?

Reply:
In Python, it goes required inputs, inputs with default/given values like  ``image_axes=image_axes``

Reply:
ok I got it